developers/index.md

```diff
@@ -124,3 +127,12 @@ This script is specific for binding addons. Follow these steps to generate your
 Now you can start developing your NEW binding. We're looking forward to your pull request!
 
 In case the new binding has dependencies to other bundles/libraries see [Adding Dependencies](buildsystem.html#adding-dependencies) for more information.
+
+If developing on Windows, git will convert line endings from LF to CRLF automatically. The repo uses LF,
+and spotless will object, and throw errors. If you use `mnv spotless:apply` to fix formatting, it will add CRLF
+to all files dirtying the entire branch. Adding a `.gitattributes` file to the openhab-addons folder with the
+following will tell git and spotless to use LF endings:
+
+```text
+* text eol=lf
+```
```

Comment:
Please also add a new line at the end.

Reply:
There is, but its not showing in the diff UI 🤷‍♂️